app/frontend/react/components/SandwichShow.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import SandwichTile from './SandwichTile'
+
+class SandwichShow extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: "",
```

Comment:
it might be more extensible to provide a `sandwich` property - that way if you add an attribute to the sandwich model, you don't have to add it in multiple places.